aws/eks/eks-how-to/deploy-ubuntu-pro-cluster.rst

```diff
@@ -0,0 +1,282 @@
+Deploy an Ubuntu Pro EKS cluster
+================================
+
+This guide shows how to deploy an EKS Cluster with Ubuntu Pro nodes.
+
+Prerequisites
+~~~~~~~~~~~~~
+
+You need:
+
+- ``eksctl``: see how to install it `here <https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html>`_
+- ``packer``: only needed if you want to enable FIPS for the cluster nodes. Install it with ``sudo snap install packer``
+- your AWS access key ID and secret access key
+- an Ubuntu Pro token
+
+
+Cluster deployment preparation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Depending on whether you need to enable FIPS or not, you only need to follow one of the following two sections.
+
+Please note that, while there are Ubuntu Pro AMIs available in AWS, at the time
+of writing this guide, there is no such offering for the EKS service, so you'll
+need to provision the EKS cluster with customized Ubuntu nodes.
+
+
+With FIPS
+^^^^^^^^^
+
+When enabling FIPS, a reboot of the underlying node is required, which if done post-cluster creation, can lead to a defective cluster.
+For this reason, the most reliable way to deploy an Ubuntu Pro EKS cluster is to actually build a
```

Comment:
I think the reason why we want to avoid a reboot is deployment speed. Not a possible bug with a reboot.

Reply:
the reboot is needed because of my comment above

Reply:
The reboot is a quick action. The long part of the cluster creation is the control plane and the provisioning of the VM.